exercise2/submissions/README.md

```diff
@@ -0,0 +1,18 @@
+Exercise 2 Notes
+================
+
+*Output files are written to the output directory and have a _comp extension.
+For instance pct.csv is replicated as pct_comp.csv in the output directory
+
+
+*I looked through the panda, numpy and scipy docs and didn't turn up a percentile function?  
+Perhaps I just missed it, but in any event the calculation is straight forward with numpy 
+array's so I did it by hand.
+
```

Comment:
I figured I was missing this.  I have updated the code/comments to use this method as well.